vassal-app/src/main/java/VASSAL/i18n/Resources.java

```diff
@@ -205,7 +217,7 @@ private void addLocale(Locale l) {
    *
    * Resources.getString(Resources.VASSAL)
    */
-  public static final String VASSAL = "General.VASSAL"; //$NON-NLS-1$
```

Comment:
Deprecate?

Reply:
Ah, that was a bug actually. I had replaced an existing in-use  key which is a no-no. Reverted, but VASSAL key moved to top of file.